tests/congruence_tests/test_search.py

```diff
@@ -144,27 +144,46 @@ def filter_search_text_single(
             limit=10,
         )
 
+    def load_query_vectors(self, query):
+        self.query_text = np.array(query.vector['text']).tolist()
+        self.query_image = np.array(query.vector['image']).tolist()
+        self.query_code = np.array(query.vector['code']).tolist()
+
```

Comment:
it's a pity, that we have to do it like this, but not sure how to make it better at the current state of the tests, either refactor more, or ditch this